fenix/app/src/main/java/org/mozilla/fenix/addons/AddonsManagementFragment.kt

```diff
@@ -107,7 +114,7 @@ class AddonsManagementFragment : Fragment(R.layout.fragment_add_ons_management)
         val allowCache = args.installAddonId == null || installExternalAddonComplete
         lifecycleScope.launch(IO) {
             try {
-                val addons = requireContext().components.addonManager.getAddons(allowCache = allowCache)
+                addons = requireContext().components.addonManager.getAddons(allowCache = allowCache)
```

Comment:
nit: I don't think you modified this but could we just say `getAddons(allowCache)`?